src/client/filehandler/FileContainer.tsx

```diff
@@ -3,16 +3,17 @@ import "./FileContainer.less";
 import { FaDownload } from "react-icons/fa";
 import { RiDeleteBin2Fill } from "react-icons/ri";
 import { useDynamicFetch } from "../hooks";
-import { FileUploadEvent } from "../../events";
-import { RoomType } from "../../types";
+import { FileUploadEvent, ResponseFormEvent } from "../../events";
+import { FileUploadType, RoomType } from "../../types";
 
 type Props = {
-    sessionID: string;
+    id: string;
     socket: SocketIOClient.Socket;
     userID: string;
-    files: Array<Array<string>>;
+    files: Array<[string, string, string, string, string, string]>;
```

Comment:
These types are highly unreadable, you should convert it to objects or at least array destruct all uses of this.

Reply:
pls bro